index.js

```diff
@@ -14,13 +14,18 @@ export class Base {
       );
     };
   }
+
   static defaults(defaults) {
     return class extends this {
       constructor(...args) {
         super(Object.assign({}, defaults, args[0] || {}));
       }
+
+      static defaultOptions = defaults;
```

Comment:
Hmm, is this actually...

```suggestion
      static defaultOptions = { ...defaults, ...super.defaults };
```

...or something similar to that. I'm poking around now.

Reply:
```suggestion
      static defaultOptions = { ...defaults, ...this.defaultOptions };
```